dlt/common/data_writers/escape.py

```diff
@@ -56,7 +67,12 @@ def escape_postgres_literal(v: Any) -> Any:
     if isinstance(v, (list, dict)):
         return _escape_extended(json.dumps(v))
     if isinstance(v, bytes):
-        return f"'\\x{v.hex()}'"
+        if is_valid_wkb(v):
```

Comment:
this is very impractical :/ we have a dependence on shapely and there may be false positives (there are no magic numbers in wkb). passing expected type to escape...literal will slow this down

we could use a special binary type but user would need to wrap binary anyway. so my take is to drop `wkb` and just keep `wkb_hex`.